Task2/index2.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Gourmet au Catering</title>
+    <link rel="stylesheet" href="style2.css">
+</head>
+<body>
+    <header>
+        <div class="container">
+            <div class="brand">
+                <h1>Gourmet au Catering</h1>
+            </div>
+            <div class="nav">
+                <ul>
+                    <li><a href="#about">About</a></li>
+                    <li><a href="#menu">Menu</a></li>
+                    <li><a href="#contact">Contact</a></li>
+                </ul>
+            </div>
+        </div>
+    </header>
+    <div class="firstsec">
+        <div class="main image">
+            <img src="images\image1.jpg" alt="Le catering">
+        </div>    
+        <h1 id="imagetext">Le Catering</h1>
+    </div> 
+    <div class="content">    
+        <div class="secondsec">
+            <div class="left">
+              <img src="images\image2.jpg" alt="About catering">
+            </div>
+            <div class="right">
+              <h2>About Catering</h2>
+              <h3>Tradition since 1889</h3>
+              <div class="catering1"><p>The Catering was founded in blabla by Mr.Smith in lorem ipsum
+                dolor sit amet, cinsectetur adipscing elit consectetur adipiscing elit,
+                sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
+                nisi ut aliquip ex ea commodo consequat. Duis aute iruredolor in 
+                reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+                pariatur.We only use <span class="highlight">seasonal</span> ingredients.</p>
+               </div>
+               <div class="catering2"><p>Excepteur sint occaecat cupidatat non proident, sunt in culpa qui 
+                officia deserunt mollit anim id est laborum consectetur adipiscing 
+                elit, sed do eiusmod temporincididunt ut labore et dolore magna
+                aliqua. Ut enim ad minim veniam, quis nostrud exerciation ullamco
+                laboris nisi ut aliquip ex ea commodo consequat. </p>
+               </div>
+            </div>
+        </div> 
+        <div class="thirdsec">
+            <div class="left">
+                <ul >
+                  <li><h2>Our Menu</h2>
```

Comment:
Also `<li><h2>Our Menu</h2>` no need of including h2 inside li. You can place h2 outside of ul tag since it is the heading of that block.